recipes/tbtamr/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "tbtamr" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "72db1eb835574f02b0d7768e6b04e310bd2229d0f684100004b20a133eecd882" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - tbtamr=tbtamr.tbtamr:main
+  run_exports:
+    - {{ pin_subpackage('tbtamr', max_pin="x.x.x") }}
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - psutil
+    - pandas
+    - csvtk
+    - bwa
+    - samtools =1.12
+    - snpeff =5.0
+    - samclip 
+    - freebayes 
+    - bcftools 
+    - bedtools 
+    - delly
+    
+test:
+  imports:
+    - tbtamr
+  commands:
+    - tbtamr --help
+
+about:
+  home: "https://github.com/MDU-PHL/tbtamr"
+  license: "GNU General Public v3 (GPLv3)"
```

```suggestion
  license: GPL-3.0-or-later
  license_file: LICENSE
```

This will only work if https://github.com/MDU-PHL/tbtamr/pull/8 gets accepted and released. But we need to ship a license with the package if it is GPL.